ipaserver/install/server/install.py

```diff
@@ -336,6 +336,12 @@ def install_check(installer):
             "Please uninstall it before configuring the IPA server, "
             "using 'ipa-client-install --uninstall'")
 
+    instance_name = "-".join(options.realm_name.split("."))
+    if (is_ipa_configured() and
```

Comment:
The correct way to check whether this is a 2nd step of external-CA installation is on (original) line 729:
```python
if not options.external_cert_files:
    ... code ...
```
I would prefer all such code is to be found at that very spot if it's not really necessary to do otherwise and I don't believe it is in this case.

I am not sure whether DS should be started in this case or not, if it were me, I would just probably have it started since it's a requirement for the installation.